src/DataProtection/Cryptography.Internal/src/CryptoUtil.cs

```diff
@@ -91,12 +101,21 @@ public static bool TimeConstantBuffersAreEqual(byte[] bufA, int offsetA, int cou
             // An error at the call site isn't usable for timing attacks.
             Assert(countA == countB, "countA == countB");
 
+#if NETCOREAPP
+            unsafe
```

Comment:
Extreme nits: No need to use unsafe here. If desired, you can also put the _NoInlining | NoOptimization_ attribute within an `#if !NETCOREAPP` block. Maybe those can come with the spanification PR so as not to cause noise here.

Reply:
Will address this in the span PR